### PR TITLE
TabView update

### PR DESCRIPTION
**Pull request by beervoley**
## Description

This PR updates the TabView. 
## How Has This Been Tested?

Manually.
## Screenshots (if appropriate):

Will attach in a bit :)

### CODE REVIEW

**Comment by tashatitova** [comment]
would be nice to have screenshots/short video

File: dev/TabView/TabView.xaml [hunk 1]
```diff
@@ -424,7 +424,7 @@
                                     <VisualState.Setters>
                                         <Setter Target="TabContainer.Background" Value="{ThemeResource TabViewItemHeaderBackgroundSelected}" />
                                         <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource TabViewItemHeaderForegroundPointerOver}" />
```
Comment (tashatitova) [inline dev/TabView/TabView.xaml:426]: looking at Figma, selected and inactive states have different pointer over and pressed states. Right now, there is the same TabViewItemHeaderForegroundPressed in Pressed and SPressedSelected which doesn't seem correct. Same for TabViewItemHeaderForegroundPointerOver

File: dev/TabView/TabView.xaml [hunk 2]
```diff
@@ -438,7 +438,7 @@
                                     <VisualState.Setters>
                                         <Setter Target="TabContainer.Background" Value="{ThemeResource TabViewItemHeaderBackgroundSelected}" />
                                         <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource TabViewItemHeaderForegroundPressed}" />
```
Comment (tashatitova) [inline dev/TabView/TabView.xaml:440]: In Selected, foreground is always the same. Would make more sense to not use TabViewItemHeaderForegroundPointerOver and Pressed for selected states but rather have a TabViewItemHeaderForegroundSelected brush, which is used for both icon and header text.

**Comment by gabrielconl** [comment]
Currently, `TabViewHeaderPadding` is set to `8,8,4,0`, which causes the rightmost corner to be cut. Maybe it could be fixed in this PR.
Also, the effective left padding is 7 because all tabs are moved 1px to the left, so it should be `9,8,8,0` to look equal.
tabview

**Comment by beervoley** [comment]
@ranjeshj yes, I'll do that :)

**Pull request closed without merging**